app/views/posts/show.html.erb

```diff
@@ -7,14 +7,18 @@
 </div>
 <div class="second">
 <p>comments: <%= @post.comments_counter %></p>
-<p> likes: <%= @post.likes_counter %></p>
+<p> likes: <%= @post.likes_counter %>
+<%= button_to "Like", user_post_likes_path(@user, @post), method: :post %>
+</p>
 </div>
 </div>
 </div>
 <ul class="comments">
 <% @comments.each do |comment| %>
     <li> <%= comment.user.name %>: <%= comment.text %> </li>
     <% end %>
+    <a href="/users/<%= @user.id %>/posts/<%= @post.id %>/comments/new">Add Comment</a>
+
 </ul>
 
```

Comment:
![image](https://github.com/AbbasSarwar/BlogApp/assets/93039359/0cb58e4d-0f81-4867-9769-dd92afc9b698)

- [ ] It would be nice if you fix the link highlighted in the image above ⬆️ because right now it is broken 😨 and taking us no where 🤔 Kindly fix it to make your readme more descriptive and professional 👍